recipe/meta.yaml

```
@@ -21,7 +21,7 @@ requirements:
     - python >=3.7
     - boto3 >=1.19.5,<2
     - jsonschema >=3.2,<5
-    - pydantic ~=1.10.2
+    - pydantic >=1.10.2,<3
```

Based on the requirements defined in the up-stream repository https://github.com/aws/serverless-application-model/blob/develop/requirements/base.txt 
```suggestion
    - pydantic >=1.8,<3
```